vscode-extensions/project-explorer/extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "vscode-project-explorer-extension",
+  "private": true,
+  "version": "11.2.0",
+  "description": "VSCode extension that provides the Axon Ivy project explorer",
```

Comment:
The Axon Ivy Project Explorer is meticulously designed to seamlessly integrate Axon Ivy projects into your development workflow.